ci-operator/step-registry/mco/conf/day2/destroy-custom-mcp/mco-conf-day2-destroy-custom-mcp-commands.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+function set_proxy () {
+    if [ -s "${SHARED_DIR}/proxy-conf.sh" ]; then
+        echo "Setting the proxy ${SHARED_DIR}/proxy-conf.sh"
+        # shellcheck source=/dev/null
+        source "${SHARED_DIR}/proxy-conf.sh"
+    else
+        echo "No proxy settings"
+    fi
+}
+
+function run_command() {
+    local CMD="$1"
+    echo "Running command: ${CMD}"
+    eval "${CMD}"
+}
+
+# waiting for mcp become a given status
+function wait_for_config_to_be_applied() {
+    local mcp_name="$1"
+    local expected_machine_count="$2"
+    local out MACHINECOUNT 
+
+    echo "Waiting for $mcp_name MachineConfigPool to finish updating..."
+    oc wait mcp "$mcp_name" --for='condition=UPDATED=True' --timeout=300s 2>/dev/null
+    
+    out="$(oc get machineconfigpools ${mcp_name} --no-headers)"
+    echo $out
+    MACHINECOUNT="$(echo "${out}" | awk '{print $6}')"
+
+    if [[ "${MACHINECOUNT}" != "${expected_machine_count}" ]]; then
+        run_command "oc get machineconfigpools"
+        run_command "oc get nodes"
+        echo "MCP ${mcp_name} is not ready for next step, exit..." && exit 1
+    fi
+}
+
+function remove_custom_mcp() {
+    local mcp_name="$1" mcp_node_label label
+    mcp_node_label=$(oc get mcp ${mcp_name} -ojson | jq -r '.spec.nodeSelector.matchLabels | to_entries | map([.key, .value] | join("=")) | join(",")')
+    local -a matched_nodes_array=()
+    local -a label_array=()
+
+    echo "Removing customer worker node label from mcp ${mcp_name}"
+    read -a matched_nodes_array <<< "$(oc get nodes -l ${mcp_node_label} -ojsonpath="{.items[:].metadata.name}")"
+    echo -e "Matched nodes with label ${mcp_node_label}: ${matched_nodes_array[*]}\n"
+
+    if [[ ${#matched_nodes_array[@]} == 0 ]]; then
+        echo "No matched nodes!"
+        return 0
+    fi
+
+    local out WORKER_MACHINECOUNT EXPECTED_WORKER_MACHINECOUNT node arrLabel
```

Comment:
`arrLabel` is supposed to be a array var, so it should be like:
```
local -a arrLabel=()
```

Reply:
Good catch, I will update it after the test job finished.

Reply:
Done